best-practices.md

```diff
@@ -561,10 +561,10 @@ just meant to give users a sense of what types of values they could expect.
 
 ### Use of links
 
-The STAC specifications allow both relative and absolute links, and says that `self` links are not required, but are 
-strongly recommended. This is what the spec must say to enable the various use cases, but there is more subtlety for when it 
-is essential to use different link types. The best practice is to use one of the below catalog types, applying the link 
-recommendations consistently, instead of just haphazardly applying relative links in some places and absolute ones in other places.
+The STAC specifications allow both relative and absolute links. This is what the spec must say to enable the various use cases, but
```

Comment:
This sentence may not make much sense now after the strongly recommended part is taken out out of the previous one?

Reply:
Yeah now that I'm rereading it I agree with you @jonhealy1. How does this sound?

```diff
diff --git a/best-practices.md b/best-practices.md
index 4373c47c..68902c78 100644
--- a/best-practices.md
+++ b/best-practices.md
@@ -561,10 +561,10 @@ just meant to give users a sense of what types of values they could expect.
 
 ### Use of links
 
-The STAC specifications allow both relative and absolute links. This is what the spec must say to enable the various use cases, but
-there is more subtlety for when it is essential to use different link types. The best practice is to use one of the below catalog
-types, applying the link recommendations consistently, instead of just haphazardly applying relative links in some places and
-absolute ones in other places.
+The STAC specifications allow both relative and absolute links, and it is important to choose the correct link types so that
+your STAC catalogs are easy to explore and resilient to any future changes to their layouts. The best practice is to use one
+of the below catalog types, applying the link recommendations consistently, instead of just haphazardly applying relative links
+in some places and absolute ones in other places.
 
 #### Self-contained Catalogs
 
```